packages/studio-desktop/src/electronBuilderConfig.js

```diff
@@ -48,7 +48,7 @@ function makeElectronBuilderConfig(params) {
         },
         {
           ext: "foxe",
-          name: "Foxbox Extension",
+          name: "Lichtblicklick Extension",
```

Comment:
"Lichtblickklick" intended?

Reply:
No, thanks for the hint. This PR is an work in progress :)

Reply:
Fixed :)